packages/docs-gesture-handler/plugins/swm-geo.js

```
@@ -116,7 +116,9 @@ module.exports = function swmGeoPlugin(context) {
           {
             tagName: 'script',
             attributes: { type: 'application/ld+json' },
-            innerHTML: JSON.stringify(buildStructuredData(context.siteConfig)),
+            innerHTML: JSON.stringify(
+              buildStructuredData(context.siteConfig),
```

ESLint

[error] Delete `,`

(prettier/prettier)